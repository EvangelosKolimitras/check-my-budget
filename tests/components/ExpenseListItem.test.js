import React from "react"
import { shallow } from "enzyme"
import expenses from "../Fixtures/expenses"
import ExpenseListItem from "../../src/scripts/js/Components/ExpenseListItem"

test("should render ExpenseListItem correctly ", () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />)
    expect(wrapper).toMatchSnapshot()
})
