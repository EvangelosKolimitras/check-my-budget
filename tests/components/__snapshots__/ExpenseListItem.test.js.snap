// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components the ExpenseListItem should render ExpenseListItem correctly  1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    0.03
    <span>
      â‚¬
    </span>
     -Created On: 
    January 1st, 1970
  </p>
</div>
`;
